beta-disclaimer.md

```diff
@@ -0,0 +1,6 @@
+---
+
+**Please note** that this is library is in a **beta** version and backwards-incompatible changes might be introduced in future releases.
```

Comment:
Sounds great overall. I would perhaps be more explicit that this could happen even in minor releases.

Reply:
Yes, I think giving some idea of breaking in minors is better. Also, I think its important that each ecosystem is able to provide clearer directions on the versioning and compatibility, as the .NET library versioning is more subtle than this sentence suggests.

Reply:
I agree with what said above, furthermore, I think this message should live in the README :)

Reply:
Fair point gentlemen. I added a part about possible minor BC's. Re-requesting approval